ampligraph/latent_features/models.py

```diff
@@ -1117,7 +1136,7 @@ def _initialize_eval_graph(self, mode="test"):
                                                                                      (None, self.internal_k))) 
 
                 corruption_generator = corruption_generator.repeat()
-                corruption_generator = corruption_generator.prefetch(1)
+                corruption_generator = corruption_generator.prefetch(0)
```

Comment:
What is this change?

Reply:
This change is actually not a part of this feature. The corruption_generator shouldn't prefetch batches for large graphs (where variables don't fit on the GPU memory)